<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>350</width>
      <height>200</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>350</width>
      <height>300</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="title">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>330</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string>Instalacja modyfikacji G2 Ucieczka</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="sourcePath">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>230</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="browseButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>90</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Przeglądaj</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxIcon">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>330</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Utwórz ikonę na pulpicie</string>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>300</width>
       <height>20</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QPushButton" name="installButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>210</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Instaluj</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelBrowse">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>330</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Wskaż folder z zainstalowanym G2NK</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxUnion">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>330</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Zainstaluj Union 1.0e</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelStatus">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>321</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
